packages/salesforcedx-vscode-lightning/package.json

```diff
@@ -71,6 +71,15 @@
     "workspaceContains:**/workspace-user.xml"
   ],
   "main": "./out/src",
+  "serverPath": [
+    "..",
+    "..",
+    "node_modules",
+    "@salesforce",
+    "aura-language-server",
+    "lib",
+    "server.js"
+  ],
```

Comment:
So the deal here is the server file lives in the root node_modules directory when running in debug mode as part of the monorepo, but lives in the extension level node_module directory when running as a vsix.  I updated both the lightning and lwc extension to account for that change as part of the packaging.  Part of this is updating where each extension gets the path to the server to follow the same pattern we see in other extensions.